<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkDiffusionQuantificationViewControls</class>
 <widget class="QWidget" name="QmitkDiffusionQuantificationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>858</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <property name="styleSheet">
   <string>QCommandLinkButton:disabled {
  border: none;
}

QGroupBox {
  background-color: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>25</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Raw diffusion-weighted image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QCommandLinkButton" name="m_MdDwiButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>MD</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCommandLinkButton" name="m_AdcDwiButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ADC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Tensor image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCommandLinkButton" name="m_FAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>FA (Fractional Anisotropy)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCommandLinkButton" name="m_RAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>RA (Relative Anisotropy)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCommandLinkButton" name="m_ADButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>AD (Axial Diffusivity)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCommandLinkButton" name="m_RDButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>RD (Radial Diffusivity)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCommandLinkButton" name="m_MDButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>MD (Mean Diffusivity)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCommandLinkButton" name="m_ClusteringAnisotropy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>1-(λ2+λ3)/(2*λ1)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="m_InputData">
     <property name="title">
      <string>Input Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Scale Image Values:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="m_ScaleImageValuesBox">
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QmitkDataStorageComboBox" name="m_ImageBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>ODF image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCommandLinkButton" name="m_GFAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>GFA</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">QmitkDataStorageComboBox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
